src/content/docs/infrastructure/host-integrations/host-integrations-list/postfix-monitoring-integration.mdx

```diff
@@ -23,83 +23,65 @@ After setting up our Postfix integration, we give you a dashboard for your Postf
 </figcaption>
 
 
-## Install the infrastructure agent [#infra]
-
-To get data into New Relic, install our infrastructure agent. Our infrastructure agent collects and ingests data so you can keep track of your mail server's performance. The version should be 1.10.7 or higher to support NRI-Flex integration.
-
-You can install the infrastructure agent two different ways:
-
-* Our [guided install](https://one.newrelic.com/nr1-core?state=4f81feab-35f7-e97e-9903-52510f8542bd) is a CLI tool that inspects your system and installs the infrastructure agent alongside the application monitoring agent that best works for your system. To learn more about how our guided install works, check out our [Guided install overview](/docs/infrastructure/host-integrations/installation/new-relic-guided-install-overview).
-* If you'd rather install our infrastructure agent manually, you can follow a tutorial for manual installation for [Linux](/docs/infrastructure/install-infrastructure-agent/linux-installation/install-infrastructure-monitoring-agent-linux), [Windows](/docs/infrastructure/install-infrastructure-agent/windows-installation/install-infrastructure-monitoring-agent-windows/), or [macOS](/docs/infrastructure/install-infrastructure-agent/macos-installation/install-infrastructure-monitoring-agent-macos/).
+## Set up the Postfix integration
```

Comment:
Intro reads a little awkward but it's not letting edit in GH:

> Our Postfix integration consolidates mail server data into one view, so you can track how many emails are sent, bounced, or deferred in a given time period. Our Postfix integration uses our infrastructure monitoring agent, and gives you a pre-built dashboard with your most important Postfix metrics like email delivery process, mail queue, and message overview.

Reply:
idk edit this so its smoother or just take it. "performance metrics" for a mail server felt a little stilted